tinytuya/core.py

```diff
@@ -479,7 +488,6 @@ def parse_header(data):
         #seqno |= unknown << 32
         total_length = payload_len + header_len + len(SUFFIX_6699_BIN)
     else:
-        #log.debug('Header prefix wrong! %08X != %08X', prefix, PREFIX_VALUE)

```

Comment:
While we are here, should we add a note?

```python
# Tuya protocol should include a header prefix 0x55AA (3.1, 3.2, 3.3, 3.4) or 0x6699 (3.5) 
# (see https://github.com/jasonacox/tinytuya/discussions/260)
```

And instead of "Header prefix wrong!" - should we error out with "Header prefix unknown!" ?  Not that Tuya would ever change that with the next protocol rev. 😉